model/Role/Role.php

```diff
@@ -206,12 +207,12 @@ public static function ODJEL_Z_LETOSNIHO_GC(int $rok = ROCNIK): int
         return self::odjelZRocniku($rok);
     }
 
-    public static function ZKONTROLOVANE_UDAJE_LETOSNIHO_GC(int $rok = ROCNIK): int
+    public static function ZKONTROLOVANE_UDAJE_PRO_LETOSNI_GC(int $rok = ROCNIK): int
```

Comment:
Tady jsem si dovolil lehké přejmenování, aby to znělo méně jako "ověření údajů pro celý Gamecon".